scipy/special/_special_ufuncs_docs.cpp

```diff
@@ -410,6 +402,83 @@ const char *binom_doc = R"(
     (0.037399983365134115, 0.0)
     )";
 
+const char *cospi_doc = R"(
+    cospi(x, out=None)
+
+    Cosine with `x` multiplied by `pi`.
+    Conforms to IEEE754-2008 specification for the sign of 0.
+
+    Parameters
+    ----------
+    x : array_like, real or complex
+        Argument
+    out : ndarray, optional
+        Optional output array for the function results.
+
+    Returns
+    -------
+    scalar or ndarray
+        Cosine at the input.
+
+    See Also
+    --------
+    sinpi, tanpi, cotpi
+
+    Examples
+    --------
+    >>> import numpy as np
+    >>> import scipy.special as sc
+
+    >>> x = np.array([-2.5, -2, -1.5, -1, -0.5, -0.0, 0.0, 0.5, 1.0, 1.5, 2, 2.5], dtype=np.float64)
+    >>> sc.cospi(x)
+    array([  0.0000000e+00,  1.0000000e+00,  0.0000000e+00, -1.0000000e+00,
+             0.0000000e+00,  1.0000000e+00,  1.0000000e+00,  0.0000000e+00,
+            -1.0000000e+00,  0.0000000e+00,  1.0000000e+00,  0.0000000e+00 ])
+    >>> np.cos(x * np.pi)
+    array([ -0.0000000e+00,  1.0000000e+00, -1.8369702e-16, -1.0000000e+00,
+             6.1232340e-17,  1.0000000e+00,  1.0000000e+00,  6.1232340e-17,
+            -1.0000000e+00, -1.8369702e-16,  1.0000000e+00,  3.0616170e-16 ])
+    )";
+
+const char *cotpi_doc = R"(
+    cotpi(x, out=None)
+
+    Cotangent with `x` multiplied by `pi`.
+    Equivalent to cospi(x)/sinpi(x).
```

Comment:
In addition to the comments for `cospi`, it may be helpful to mention in the "Notes" section (1) at what inputs the function returns `+/-inf` and whether the function sets a warning in this case, (IEEE's tanpi recommends "divideByZero" exception in this scenario,) and (2) that the function may overflow for subnormal input $x$ and whether an overflow warning is set in this case.